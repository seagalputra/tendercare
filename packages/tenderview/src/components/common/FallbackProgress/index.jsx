import React from 'react'
import { CircularProgress, Container } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles(() => ({
  progress: {
    textAlign: 'center'
  }
}))

const FallbackProgress = () => {
  const classes = useStyles()

  return (
    <Container className={classes.progress}>
      <CircularProgress />
    </Container>
  )
}

export default FallbackProgress
