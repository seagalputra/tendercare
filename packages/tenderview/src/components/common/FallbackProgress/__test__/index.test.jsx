import React from 'react'
import { shallow } from 'enzyme'
import FallbackProgress from '..'

describe('FallbackProgress', () => {
  it('renders without crashing', () => {
    const component = shallow(<FallbackProgress />)
    expect(component).toMatchSnapshot()
  })
})
