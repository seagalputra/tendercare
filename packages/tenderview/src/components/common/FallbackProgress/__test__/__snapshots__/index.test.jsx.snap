// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FallbackProgress renders without crashing 1`] = `
<WithStyles(ForwardRef(Container))
  className="makeStyles-progress-1"
>
  <WithStyles(ForwardRef(CircularProgress)) />
</WithStyles(ForwardRef(Container))>
`;
